tests/playwright/src/specs/extension-installation.spec.ts

```diff
@@ -73,60 +69,59 @@ beforeEach<RunnerTestContext>(async ctx => {
 
 describe.each([
   {
-    extensionType: 'Developer Sandbox',
+    extensionType: 'Red Hat OpenShift Sandbox',
   },
   {
-    extensionType: 'Openshift Local',
+    extensionType: 'Red Hat Openshift Local',
   },
 ])('$extensionType installation verification', async ({ extensionType }) => {
   beforeAll(_startup);
   afterAll(_shutdown);
 
   test('Initialize extension type', async () => {
     initializeLocators(extensionType);
-    pdRunner.setVideoAndTraceName(`${extensionName}-installation-e2e`);
-    extensionCard = new ExtensionCardPage(page, extensionLabel, extensionName);
-    extensionBoxVisible = await extensionDashboardBox.isVisible();
+    await navBar.openExtensions();
   });
 
-  describe('Check installation availability', async () => {
-    test.runIf(extensionBoxVisible)('Check Dashboard extension component for installation availability', async () => {
-      const installButton = extensionDashboardBox.getByRole('button', { name: installButtonLabel });
-      await playExpect(installButton).toBeVisible();
-    });
+  test('Install extension through Extensions Catalog', async () => {
+    const extensionsPage = new ExtensionsPage(page);
 
-    test('Go to extensions', async () => {
-      await navBar.openExtensions();
-    });
+    await extensionsPage.openCatalogTab();
+    const extensionCatalog = new ExtensionCatalogCardPage(page, extensionType);
+    await playExpect(extensionCatalog.parent).toBeVisible();
 
-    // requires to be rewritten, as extension are not shuffled anymore, but are part of extensions catalog
-    test.runIf(extensionBoxVisible)('Check Settings extension component for installation availability', async () => {
-      const installButton = extensionCard.card.getByRole('button', { name: installButtonLabel });
-      await playExpect(installButton).toBeVisible();
-    });
-  });
+    await playExpect.poll(async () => await extensionCatalog.isInstalled(), { timeout: 5000 }).toBeFalsy();
```

Comment:
Default timeout for waits is 5 seconds, so there is no reason to add it explicitly except if you want to set a different value.

Reply:
nice.